cerberus/client.py

```diff
@@ -76,11 +80,11 @@ def _set_token(self):
         try:
             self.token = os.environ['CERBERUS_TOKEN']
             if self.verbose:
-                print("Overriding Cerberus token with environment variable.", file=sys.stderr)
+                logger.debug("Overriding Cerberus token with environment variable.", file=sys.stderr)
             logger.info("Overriding Cerberus token with environment variable.")
             return
-        except:
-            pass
+        except Exception as e:
+            logger.debug(str(e))
```

Comment:
Did this print the stack trace before and will now debug log it if debug is enabled?

- [ ] Ensure that the stack trace is still logged by default, if that was the previous behavior.